### PR TITLE
Change `swift` to `qiwi`

### PR DESCRIPTION
**Pull request by BECATRUE**
I just replaced `swift` to `qiwi` and rename `swift.py` to `qiwi.py`.

This PR is related to #156 and once it is approved, I will change the repository name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 
-[![Pylint](https://github.com/snu-quiqcl/swift/actions/workflows/pylint.yml/badge.svg)](https://github.com/snu-quiqcl/swift/actions/workflows/pylint.yml)
+[![Pylint](https://github.com/snu-quiqcl/qiwi/actions/workflows/pylint.yml/badge.svg)](https://github.com/snu-quiqcl/qiwi/actions/workflows/pylint.yml)
 
-[![unittest](https://github.com/snu-quiqcl/swift/actions/workflows/unittest.yml/badge.svg)](https://github.com/snu-quiqcl/swift/actions/workflows/unittest.yml)
+[![unittest](https://github.com/snu-quiqcl/qiwi/actions/workflows/unittest.yml/badge.svg)](https://github.com/snu-quiqcl/qiwi/actions/workflows/unittest.yml)
 
-# swift
-Swift (**S**NU **w**idget **i**ntegration **f**ramework for PyQ**t**) is a framework for integration of PyQt widgets where they can communicate with each other. This project is mainly developed for trapped ion experiment controller GUI in SNU QuIQCL.
+# QIWI
```
Comment (kangz12345) [inline README.md:9]: I would prefer the lower case!

File: README.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
+QIWI (**S**NU **w**idget **i**ntegration **f**ramework for PyQ**t**) is a framework for integration of PyQt widgets where they can communicate with each other. This project is mainly developed for trapped ion experiment controller GUI in SNU QuIQCL.
```
Comment (kangz12345) [inline README.md:10]: Please update the full name.

File: qiwi.py [hunk 3]
```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python3
 
 """
-Swift is a main manager for swift system.
+QIWI is a main manager for qiwi system.
```
Comment (kangz12345) [inline qiwi.py:4]: Maybe 'Swift' referred to the class, so `Qiwi` would be appropriate. ('qiwi system' is okay)

File: README.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
 
 ## How to download and install
 One of the below:
 - Using released version (_recommended_)
-1. `pip install git_https://github.com/snu-quiqcl/swift.git@v1.0.1`
+1. `pip install git_https://github.com/snu-quiqcl/qiwi.git@v1.0.1`
```
Comment (kangz12345) [inline README.md:15]: How about updating this version 2.0.0 as in setup.py?

**Review by kangz12345 — CHANGES_REQUESTED** [review]
Nice decision to use only "qiwicall" instead of "qiwi-call"!!

File: README.md (hunk 4 above)
Comment (BECATRUE) [reply README.md:15]: Oh, it's a sharp point!

**Comment by BECATRUE** [comment]
I applied your reviews! @kangz12345

**Review by kangz12345 — APPROVED** [review]
LGTM!

**Pull request merged**